mailpoet/lib/EmailEditor/Integrations/Core/Renderer/Blocks/Column.php

```diff
@@ -30,19 +30,19 @@ private function prepareColumnTemplate(array $parsedBlock): string {
     $paddingTop = $parsedBlock['attrs']['style']['spacing']['padding']['top'] ?? '0px';
 
     return '
-      <!--[if mso | IE]><td class="" style="vertical-align:top;width:' . $width . ';"><![endif]-->
-      <div class="email_column" style="background:' . $backgroundColor . ';background-color:' . $backgroundColor . ';width:100%;max-width:' . $width . ';font-size:0px;text-align:left;display:inline-block;vertical-align:top;">
-        <table class="email_column" border="0" cellpadding="0" cellspacing="0" role="presentation" style="background:' . $backgroundColor . ';background-color:' . $backgroundColor . ';width:100%;max-width:' . $width . ';vertical-align:top;" width="' . $width . '">
-          <tbody>
-            <tr>
-              <td align="left" style="font-size:0px;padding-left:' . $paddingLeft . ';padding-right:' . $paddingRight . ';padding-bottom:' . $paddingBottom . ';padding-top:' . $paddingTop . ';">
-                <div style="line-height:1;text-align:left;">{column_content}</div>
-              </td>
-            </tr>
-          </tbody>
-        </table>
-      </div>
-      <!--[if mso | IE]></td><![endif]-->
+      <td class="block" style="vertical-align:top;width:' . $width . ';background:' . $backgroundColor . ';background-color:' . $backgroundColor . ';">
+        <div class="email_column" style="background:' . $backgroundColor . ';background-color:' . $backgroundColor . ';width:100%;max-width:' . $width . ';font-size:0px;text-align:left;display:inline-block;vertical-align:top;">
```

Comment:
Could you please add more details about why this change was needed? Maybe also to the commit message.
 I wonder if we need to enable the table cell for all email clients. What if we move te `block` class to the div inside?

Reply:
I'm sorry for your confusion. I added another table wrapper to unify the height of the `core/column` blocks inside a `core/columns`. For example, when you set a different background for each column, the height of the block depends on the content inside. This change fixes this issue.
[Here](https://d.pr/i/eu7Dz3) is an image containing a comparison.

Reply:
Thank you for the explanation.